src/Wrido.Web/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "wrido",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-redux": "^5.0.6",
+    "react-scripts": "1.1.0",
+    "redux": "^3.7.2",
+    "redux-observable": "^0.17.0",
+    "reset-css": "^2.2.1",
+    "rxjs": "^5.5.6"
```

Comment:
You should be able to add [`@aspnet/signalr-client`](https://www.npmjs.com/package/@aspnet/signalr-client) as a dependency here (and remove the file from _src/Wrido.Web/public/signalr-client-1.0.0-alpha2-final.js_)

Reply:
Yes, wil fix.